quickstart/deploy-mayastor.md

```diff
@@ -10,6 +10,12 @@ The steps and commands which follow are intended only for use in conjunction wit
 
 ## Installation via helm
 
+{% hint style="info" %}
+Installing Mayastor via the Helm chart sets the storageClass parameters, both for Loki's central log storage and for etcd's StatefulSets, as defaults. To provision a hostPath PV, make the following changes in the values.yaml file:
```

Comment:
Installing Mayastor via the Helm chart sets the StorageClass as 'default' for Loki and etcd statefulsets. This will work in clusters which have a StorageClass named 'default'. If there is no 'default' SC in the cluster, storage provisioning will fail for Loki and etcd. In such cases, one can change the SC to 'manual' which will provision a PersistentVolume of type hostPath. To do this, make the following changes in the values.yaml file:

Reply:
one question: is sc name `default` compulsory ? or any name storage class will fine but that sc should be default class in cluster with `storageclass.kubernetes.io/is-default-class":"true"`

Reply:
@w3aman it's a named storage class, just that the name is 'default'. Usually, all managed K8s offerings have a SC named 'default'.